hw/ip/otbn/dv/otbnsim/sim/wsr.py

```diff
@@ -13,6 +15,10 @@
 DEBUG_KMAC = False
 
 
+def eprint(text):
+    print(text, file=sys.stderr)
```

Comment:
nit: Since it looks to me like all of these `eprint`s are gated by `DEBUG_KMAC`, maybe it would make sense to do:
```suggestion
def kmac_debug_print(text):
   if DEBUG_KMAC:
      print(text, file=sys.stderr)
```
...and then remove several `if DEBUG_KMAC` clauses later on.

Reply:
Great idea! I had needed to change the debug print to eprint when debugging RTL since it other wise conflicts with the DV environment trying to send its trace data over stdout instead of stderr